helpdesk_mgmt_ticket_stage_auto/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Helpdesk Ticket Stage Auto",
+    "summary": """
+        Adds a programmed action to change the auto stage
```

Comment:
Maybe a better description should be the following:
```suggestion
        Enables automatic stage change for tickets due to inactivity
```